virtual-repeater.js

```diff
@@ -344,10 +339,8 @@ export const Repeats = Superclass => class extends Superclass {
     const end = this._last;
     for (let idx = start; idx <= end; idx++) {
       const child = this._assignChild(idx);
-      if (this._manageDom) {
-        if (this._maintainDomOrder || !this._childIsAttached(child)) {
-          this._insertBefore(child, null);
-        }
+      if (!this._childIsAttached(child)) {
```

Comment:
argh, here we should have removed the if check completely in order to keep the dom order